src/styles/utilities/_responsive-embed.scss

```diff
@@ -0,0 +1,19 @@
+.responsive_embed {
```

Comment:
Couple of things. This doesn’t feel like a utility to me. It’s more of a component. The selector should be kebab-case, and should start with the letter that denotes the kind of selector it is. If this stays in utilities it would be `u-`, and for components it’s `c-`.

This is the kind of thing I’m going to make sure is better documented in future :)